_run/kube/README.md

```
@@ -41,250 +32,224 @@ this example.
 
-Get service status
+#### Check Status of  Deployment
 
-__t1 service status__
-```sh
+```
 make provider-lease-status
 ```
 
-Fetch logs from deployed service (all pods)
+#### Ping the Deplpyment to Ensure Liveness
```

markdownlint-cli2 (0.20.0)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)